irods/access.py

```diff
@@ -45,10 +63,31 @@ def __init__(self, access_name, path, user_name='', user_zone='', user_type=None
         self.user_zone = user_zone
         self.user_type = user_type
 
+    def copy(self, decanonicalize = False):
+        other = copy.deepcopy(self)
+        if decanonicalize:
+            replacement_string = { 'read object':'read',
+                                   'read_object':'read',
+                                   'modify object':'write',
+                                   'modify_object':'write'}.get(self.access_name)
+            other.access_name = replacement_string if replacement_string is not None \
+                                else self.access_name
+        return other
+
     def __repr__(self):
         object_dict = vars(self)
         access_name = self.access_name.replace(' ','_')
         user_type_hint = ("({user_type})" if object_dict.get('user_type') is not None else "").format(**object_dict)
         return "<iRODSAccess {0} {path} {user_name}{1} {user_zone}>".format(access_name,
                                                                             user_type_hint,
                                                                             **object_dict)
+
+class _iRODSAccess_4_2(iRODSAccess):
+    codes = collections.OrderedDict(
+        (key.replace('_',' '),value) for key,value in iRODSAccess.codes.items() if key in (
+            'own',
+            'write', 'modify_object',
```

Comment:
Should these have spaces instead of underscores?

Reply:
no?    this is the new thing...

Reply:
Oh, it helps when you read the code. It's replacing the underscores with spaces on 87. Please disregard